modules/calib3d/src/dls.h

```diff
@@ -5,9 +5,6 @@
 
 #include <iostream>
 
-using namespace std;
-using namespace cv;
```

Comment:
Lets keep the patch minimal.
Please use `namespace cv {` here instead of `cv::` prefixes below. The same namespace cc should be used in dls.cpp file (to prevent unwanted symbols in global namespace).

It is fine to use "std::" prefixes in code.

Reply:
@alalek, I added `namespace cv {...}` and removed all `cv::` for consistency.